<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="graphLayout"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="5">
        <widget class="QComboBox" name="comboBoxT02">
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="15">
        <widget class="QComboBox" name="comboBoxT05">
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="9">
        <widget class="QComboBox" name="comboBoxT08">
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T01">
         <property name="text">
          <string>T01</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T07">
         <property name="text">
          <string>T07</string>
         </property>
        </widget>
       </item>
       <item row="0" column="10" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T04">
         <property name="text">
          <string>T04</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T06">
         <property name="text">
          <string>T06</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T02">
         <property name="text">
          <string>T02</string>
         </property>
        </widget>
       </item>
       <item row="0" column="13" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T05">
         <property name="text">
          <string>T05</string>
         </property>
        </widget>
       </item>
       <item row="2" column="7" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T08">
         <property name="text">
          <string>T08</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBoxT01">
         <item>
          <property name="text">
           <string>yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox3">
         <property name="text">
          <string>Grid Vertical</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBoxT06">
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox2">
         <property name="text">
          <string>Grid Horizontal</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="T03">
         <property name="text">
          <string>T03</string>
         </property>
        </widget>
       </item>
       <item row="2" column="13" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="P02">
         <property name="text">
          <string>P02</string>
         </property>
        </widget>
       </item>
       <item row="2" column="10" alignment="Qt::AlignRight">
        <widget class="QCheckBox" name="P01">
         <property name="text">
          <string>P01</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QComboBox" name="comboBoxT03">
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QComboBox" name="comboBoxT07">
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="12">
        <widget class="QComboBox" name="comboBoxT04">
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="12">
        <widget class="QComboBox" name="comboBoxP01">
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="15">
        <widget class="QComboBox" name="comboBoxP02">
         <item>
          <property name="text">
           <string>purple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lime</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deeppink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>white</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
